engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/MainMenuScreen.java

```diff
@@ -27,6 +27,8 @@
 import org.terasology.rendering.nui.widgets.UILabel;
 import org.terasology.version.TerasologyVersion;
 
+import javax.print.attribute.standard.Severity;
```

Comment:
The enum you're using is intended for a printer API and currently _happens_ to contain the three values we need, but can be arbitrarily changed in the future. Can you replace it with a custom enum located within the `CrashReporter` so that we "own" the code?